installers/msi-language/Product.p.wxs

```diff
@@ -213,11 +222,16 @@
 		<CustomAction
 			Id="SetStateDeployPropertyValues"
 			Property="StateDeploy"
-			Value="INSTALLDIR=[INSTALLDIR];PROJECT_OWNER_AND_NAME=[PROJECT_OWNER_AND_NAME];IS_MODIFY=[IS_MODIFY];STATE_TOOL_INSTALLED=[STATE_TOOL_INSTALLED];STATE_TOOL_PATH=[STATE_TOOL_PATH];AS_USERNAME=[AS_USERNAME];AS_PASSWORD=[AS_PASSWORD];AS_TOTP=[AS_TOTP];UI_LEVEL=[UILevel];MSI_VERSION=[MSI_VERSION];PRODUCT_VERSION=[ProductVersion];MsiLogFileLocation=[MsiLogFileLocation];PRIVACY_ACCEPTED=[PRIVACY_ACCEPTED];ERROR=[ERROR];ERROR_MESSAGE=[ERROR_MESSAGE];PRODUCT_NAME=[ProductName];USERSID=[UserSID];COMMIT_ID=[CommitID];INSTALL_MODE=[INSTALL_MODE]"
+			Value="INSTALLDIR=[INSTALLDIR];PROJECT_OWNER_AND_NAME=[PROJECT_OWNER_AND_NAME];IS_MODIFY=[IS_MODIFY];STATE_TOOL_INSTALLED=[STATE_TOOL_INSTALLED];STATE_TOOL_PATH=[STATE_TOOL_PATH];AS_USERNAME=[AS_USERNAME];AS_PASSWORD=[AS_PASSWORD];AS_TOTP=[AS_TOTP];UI_LEVEL=[UILevel];MSI_VERSION=[MSI_VERSION];PRODUCT_VERSION=[ProductVersion];MsiLogFileLocation=[MsiLogFileLocation];PRIVACY_ACCEPTED=[PRIVACY_ACCEPTED];ERROR=[ERROR];ERROR_MESSAGE=[ERROR_MESSAGE];PRODUCT_NAME=[ProductName];USERSID=[UserSID];COMMIT_ID=[CommitID];INSTALL_MODE=[INSTALL_MODE];LAST_DIALOG=[LAST_DIALOG]"
 		/>
 		<CustomAction Id="SetUninstallProperties" Property="Uninstall" Value="PRESET=[PRESET];REMEMBER=[REMEMBER];REMEMBER_SHORTCUTDIR=[REMEMBER_SHORTCUTDIR];MSI_VERSION=[MSI_VERSION];MsiLogFileLocation=[MsiLogFileLocation];PRIVACY_ACCEPTED=[PRIVACY_ACCEPTED]" />
-		<CustomAction Id="SetPresetInfo" Property="Preset" Value="REL_NOTES=[REL_NOTES];INSTALLDIR=[INSTALLDIR];APP_START_MENU_PATH=[ApplicationProgramsFolder];PRESET=[PRESET];MSI_VERSION=[MSI_VERSION];MsiLogFileLocation=[MsiLogFileLocation];PRIVACY_ACCEPTED=[PRIVACY_ACCEPTED]" />
-		<CustomAction Id="SetRollbackPropertyValues" Property="Rollback" Value="STATE_TOOL_INSTALLED=[STATE_TOOL_INSTALLED];INSTALLDIR=[INSTALLDIR];MSI_VERSION=[MSI_VERSION];MsiLogFileLocation=[MsiLogFileLocation];PRIVACY_ACCEPTED=[PRIVACY_ACCEPTED]" />
+		<CustomAction Id="SetPresetInfo" Property="Preset" Value="REL_NOTES=[REL_NOTES];INSTALLDIR=[INSTALLDIR];APP_START_MENU_PATH=[ApplicationProgramsFolder];PRESET=[PRESET];MSI_VERSION=[MSI_VERSION];MsiLogFileLocation=[MsiLogFileLocation];PRIVACY_ACCEPTED=[PRIVACY_ACCEPTED];LAST_DIALOG=[LAST_DIALOG];INSTALL_MODE=[INSTALL_MODE]" />
+		<CustomAction Id="SetRollbackPropertyValues" Property="Rollback" Value="STATE_TOOL_INSTALLED=[STATE_TOOL_INSTALLED];INSTALLDIR=[INSTALLDIR];MSI_VERSION=[MSI_VERSION];MsiLogFileLocation=[MsiLogFileLocation];PRIVACY_ACCEPTED=[PRIVACY_ACCEPTED];LAST_DIALOG=[LAST_DIALOG];INSTALL_MODE=[INSTALL_MODE]" />
+		<CustomAction
+			Id="SetCancelPropertyValues"
+			Property="CustomUserExit"
+			Value="INSTALL_MODE=[INSTALL_MODE];PRODUCT_VERSION=[ProductVersion];MsiLogFileLocation=[MsiLogFileLocation];LAST_DIALOG=[LAST_DIALOG]"
+		/>
```

Comment:
This custom action is created but never scheduled. You would need this if `CustomUserExit` was deferred but it's not